release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2874,6 +2874,62 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when you attempted to use {olm-first} to upgrade an Operator, the upgrade was blocked and an `error validating existing CRs against new CRD's schema` message was generated. An issue existed with {olm}, whereby {olm} erroneously identified incompatibility issues validating existing custom resources (CRs) against the new Operator version's custom resource definitions (CRDs). With this release, the validation is corrected so that Operator upgrades are no longer blocked. (link:https://issues.redhat.com/browse/OCPBUGS-46054[*OCPBUGS-46054*]) 
+
+* Previously, a `PipelineRuns` CR that used a resolver could not be rerun on the {product-title} web console. If you attempted ro rerun the CR, a `Invalid PipelineRun configuration, unable to start Pipeline.` was generated. With this release, you can now rerun a `PipelineRuns` CR that uses resolver without experience this issue. (link:https://issues.redhat.com/browse/OCPBUGS-45949[*OCPBUGS-45949*]) 
+
+* Previously, when you used the *Form View* to edit `Deployment` or `DeploymentConfig` API objects on the {product-title} web console, duplicate `ImagePullSecrets` parameters existed in the YAML configuration for either object. With this release, a fix ensures that duplicate `ImagePullSecrets` parameters do not get automatically added for either object. (link:https://issues.redhat.com/browse/OCPBUGS-45948[*OCPBUGS-45948*])
+
+* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` now includes a default region so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-45938[*OCPBUGS-45938*])
+
+* Previously, the installation of an {aws-short} cluster failed when the service control policy (SCP) had `false` specified for its `AssociatePublicIpAddress` parameter. With this release, a fix ensures that this SCP configuration no longer causes the installation of an {aws-short} cluster to fail. (link:https://issues.redhat.com/browse/OCPBUGS-45186[*OCPBUGS-45186*])
+
+* Previously, an additional filtering property was passed into the component that is used to list operands on the *Operator details* page. The additional property caused the list to always be empty if it was extended by a dynamic plugin. With this release, the extra property has been removed so that available operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-45667[*OCPBUGS-45667*])
+
+* Previously, when you ran the `oc adm node-image create` command, the command would sometimes command fail and output an `image can't be pulled` error message. With this release, a fix adds a retry mechanism to the command so that if the command fails to pull images from a release workload, a retry operation ensures the command runs as expected. (link:https://issues.redhat.com/browse/OCPBUGS-45517[*OCPBUGS-45517*])
```

Comment:
```suggestion
* Previously, when you ran the `oc adm node-image create` command, the command would sometimes fail and output an `image can't be pulled` error message. With this release, a fix adds a retry mechanism to the command so that if the command fails to pull images from a release workload, a retry operation ensures the command runs as expected. (link:https://issues.redhat.com/browse/OCPBUGS-45517[*OCPBUGS-45517*])
```

Reply:
:heavy_check_mark: